elyra/pipeline/tests/test_airflow_processor.py

```diff
@@ -0,0 +1,239 @@
+                                      metadata=image_one
+                                      ),
+                             Metadata(name="test-image-metadata",
+                                      display_name="test-image",
+                                      schema_name="airflow",
+                                      metadata=image_two
+                                      )
+                             ]
+
+    return mocked_runtime_images
+
+
+@pytest.fixture
+def parsed_ordered_dict(monkeypatch, processor, parsed_pipeline,
+                        sample_metadata, sample_image_metadata):
+
+    mocked_runtime = Metadata(name="test-metadata",
+                              display_name="test",
+                              schema_name="airflow",
+                              metadata=sample_metadata
+                              )
+
+    mocked_func = mock.Mock(return_value="default", side_effect=[mocked_runtime, sample_image_metadata])
+
+    monkeypatch.setattr(processor, "_get_metadata_configuration", mocked_func)
+    monkeypatch.setattr(processor, "_upload_dependencies_to_object_store", lambda x, y, z: True)
+    monkeypatch.setattr(processor, "_get_dependency_archive_name", lambda x: True)
+
+    return processor._cc_pipeline(parsed_pipeline, pipeline_name="some-name")
+
+
+def test_processor_type(processor):
+    assert processor.type == "airflow"
+
+
+def test_fail_processor_type(processor):
+    with pytest.raises(Exception):
+        assert processor.type == "kfp"
+
+
+def test_pipeline_process(monkeypatch, processor, parsed_pipeline, sample_metadata):
+
+    mocked_runtime = Metadata(name="test-metadata",
+                              display_name="test",
+                              schema_name="airflow",
+                              metadata=sample_metadata
+                              )
+    mocked_path = "/some-placeholder"
+
+    monkeypatch.setattr(processor, "_get_metadata_configuration", lambda namespace, name: mocked_runtime)
+    monkeypatch.setattr(processor, "create_pipeline_file",
+                        lambda pipeline, pipeline_export_format, pipeline_export_path, pipeline_name: mocked_path)
+
+    monkeypatch.setattr(github.Github, "get_repo", lambda x, y: True)
+    monkeypatch.setattr(git.GithubClient, "upload_dag", lambda x, y, z: True)
+
+    response = processor.process(pipeline=parsed_pipeline)
+
+    assert response.run_url == sample_metadata['api_endpoint']
+    assert response.object_storage_url == sample_metadata['cos_endpoint']
+    # Verifies that only this substring is in the storage path since a timestamp is injected into the name
+    assert "/" + sample_metadata['cos_bucket'] + "/" + "untitled" in response.object_storage_path
+
+
+def test_create_file(monkeypatch, processor, parsed_pipeline, parsed_ordered_dict, sample_metadata):
+    correct_hash_value = '9a15eba3337ba3c90457d5b60495720333f944fd2cec8e4ce40c32238d2a4206'
```

Comment:
Could you add a comment indicating how this hash value is computed so folks making changes know what to do?

Frankly. I'd prefer we validate the export w/o this dependency, perhaps by looking for expected values in the output python file - or things like that.  This hash stuff has proven to be quite fragile and was abandoned in [kfp-notebook](https://github.com/elyra-ai/kfp-notebook/pull/68).

Reply:
+1 abandoning the hash stuff. Ill refactor to verify via expected values